core/templates/dev/head/pages/exploration_editor/RouterService.js

```diff
@@ -153,6 +160,9 @@ oppia.factory('RouterService', [
       },
       isLocationSetToNonStateEditorTab: function() {
         var currentPath = $location.path();
+        if (currentPath === '/issues' && isIssuesTabExposed()) {
```

Comment:
Nit (optional): maybe inline this with the ``return`` statement below. That way, when the gating code is removed, it will look just like the other conditions in that statement.